service-binding/postgresql-crunchy-classic/src/test/java/io/quarkus/ts/sb/postgresql/OpenShiftPostgreSqlSbIT.java

```diff
@@ -24,6 +25,7 @@
 import io.quarkus.test.utils.Command;
 
 @OpenShiftScenario(deployment = OpenShiftDeploymentStrategy.UsingOpenShiftExtensionAndDockerBuildStrategy)
+@DisabledIfSystemProperty(named = "ts.s390x.missing.services.excludes", matches = "true", disabledReason = "Crunchy Postgres operator not available on s390x.")
```

Comment:
`Crunchy Postgres operator not available on s390x.`

I see, alright so previously you didn't know about it? Because I'm just surprised you have these problems now. We have this test for a while now, right?

Reply:
I think we when last run the test, it was already disabled. We only just saw it now since, the test was fixed not too long ago.

Reply:
I probably should have added this in my previous [PR](https://github.com/quarkus-qe/quarkus-test-suite/pull/1431)